src/krishna/Assignment_13/VowelExample.java

```diff
@@ -0,0 +1,30 @@
+package krishna.Assignment_13;
+
+public class VowelExample {
+
+	public void vowelIdentify(char ch) {
+		switch (ch) {
+		
+		case 'L':
```

Comment:
This is not I am expecting @krishnasawner 
Check the requirement again 

let say tomorrow if requirement comes for 's' char then you have to write code again.